### PR TITLE
Migrate remaining direct file writes to docker operations

### PR DESCRIPTION
**Pull request by mark-rushakoff**
I think this is all the outstanding os.WriteFile and os.ReadFile calls.
This should enable switching the host mounts to docker volumes in an
upcoming commit.

For #200.

### CODE REVIEW

File: chain/internal/tendermint/tendermint_node.go [hunk 1]
```diff
@@ -137,25 +138,18 @@ func (tn *TendermintNode) PrivValKeyFilePath() string {
...
-}
-
 // Bind returns the home folder bind point for running the node
 func (tn *TendermintNode) Bind() []string {
 	return []string{fmt.Sprintf("%s:%s", tn.Dir(), tn.HomeDir())}
 }
 
 func (tn *TendermintNode) HomeDir() string {
-	return filepath.Join("/tmp", tn.Chain.Config().Name)
+	return path.Join("/tmp", tn.Chain.Config().Name)
```
Comment (DavidNix) [inline chain/internal/tendermint/tendermint_node.go:147]: Just curious: Why the change from `filepath` to `path`? Docker volumes always `/`?

**Review by DavidNix — APPROVED** [review]

File: chain/internal/tendermint/tendermint_node.go (hunk 1 above)
Comment (mark-rushakoff) [reply chain/internal/tendermint/tendermint_node.go:147]: Correct, within the Docker volumes, we can rely on forward slashes for paths.
